providers/bulkSms/src/lib/bulkSms.provider.spec.ts

```diff
@@ -0,0 +1,3 @@
+import { BulkSmsSmsProvider } from './bulkSms.provider';
+
+test('should trigger bulkSms library correctly', async () => {});
```

Comment:
Can we add some tests here?

Reply:
Yes we can add tests here

Reply:
This hasn't been addressed yet.

Reply:
@p-fernandez have added a test which tells about the working model of bulksms

Reply:
Test is not showing up.